tools/data_scripts/fetch_aws_models.sh

```diff
@@ -33,22 +33,28 @@ MODELS_DIRNAME=models_folder
 
 cd $ROOTDIR
 
-echo "====== Downloading http://craftassist.s3-us-west-2.amazonaws.com/pubr/${MODELS_DIRNAME}_${CHECKSUM}.tar.gz to $ROOTDIR/${MODELS_DIRNAME}_${CHECKSUM}.tar.gz ======"
-curl http://craftassist.s3-us-west-2.amazonaws.com/pubr/${MODELS_DIRNAME}_${CHECKSUM}.tar.gz -o $MODELS_DIRNAME.tar.gz 
+if [ $3 == "nsp" ]; then
```

Comment:
Not sure I understand why we are introducing this ^
What is the downside of downloading nsp file by default for both agents and downloading the locobot perception files in : `agent/locobot/model/perception` when arg1 is 'locobot' ?

Reply:
This is an implementation detail about how `fetch_aws_models.sh` is called. Roughly, the try_download.sh script calls the download script twice for locobot ([here](https://github.com/facebookresearch/droidlet/blob/anurag/nsp_playground/tools/data_scripts/try_download.sh#L74) and [here](https://github.com/facebookresearch/droidlet/blob/anurag/nsp_playground/tools/data_scripts/try_download.sh#L82)). This check just ensures that the second time the script is called, the nsp models aren't re-downloaded the second time. We still download the nsp model by default for both agents.